<launch>
  <arg name="robot_name" default="$(optenv ROBOT_NAME pi_puck)" />
  <arg name="tf_rate" default="10" />

  <group ns="$(arg robot_name)">
    <param name="robot_description" textfile="$(find pi_puck_driver)/urdf/pi_puck_urdf.xml"/>
    <param name="tf_prefix" value="$(arg robot_name)"/>
    <node name="tf_broadcaster" pkg="pi_puck_driver" type="transform_server.py" output="screen">
      <param name="use_imu" value="false"/>
      <param name="use_hybrid_position" value="false"/>
      <param name="rate" value="$(arg tf_rate)" />
      <param name="publish_static_transforms" value="false" />
    </node>
   <node pkg="robot_state_publisher" type="robot_state_publisher" name="static_transform_publisher" />
  </group>
</launch>
